workflows/README.md

```diff
@@ -1 +1,58 @@
+# Getting started with Docker
```

Comment:
Both this PR and PR #49 modify this file, but the changes are not in conflict. Your modifications are adding detailed step-by-step instructions for getting started into the readme, while my modifications added high-level descriptions of different workflows taken from the project report.